modules/core/src/main/scala/net/message/StartupMessage.scala

```diff
@@ -19,16 +25,15 @@ case class StartupMessage(user: String, database: String) extends UntaggedFronte
 }
 
 object StartupMessage {
-
-  val ConnectionProperties: List[(String, String)] =
-    List(
+  val DefaultConnectionParameters: Map[String, String] =
```

Comment:
Now that this is part of the public API let's move `DefaultConnectionParameters` to the `Session` companion so users don't need to dig into the protocol implementation to find it.